tests/e2e/wdio.conf.js

```diff
@@ -0,0 +1,27 @@
+import path from 'node:path';
+import url from 'node:url';
+
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
+
+const DEBUG = process.env.DEBUG;
+
+export const config = {
+  specs: ['specs/**/*.js'],
+  capabilities: [
+    {
+      browserName: 'chrome',
+      'goog:chromeOptions': {
+        args: (DEBUG ? [] : ['headless', 'disable-gpu']).concat([
+          `--load-extension=${path.join(__dirname, '..', '..', 'dist')}`,
+          '--disable-search-engine-choice-screen',
+        ]),
+      },
+    },
+  ],
+  injectGlobals: false,
```

Comment:
what does this do?

Reply:
Turns off injecting the `browser`, `$$`, and `$` into the global contexts of specs. I prefer to import them explicitly.

Reply:
very good, that's also my preference. I was confused as you still have global `describe`.

Reply:
This is how mocha works, I am not sure if we can import `describe`, `it` and others directly.